Text/CustomEntitySearch/WordLinker.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace AzureCognitiveSearch.PowerSkills.Text.CustomEntitySearch
+{
+    class WordsConfig
+    {
+        public List<string> TargetWords { get; set; }
+    }
+    class WordLinker
```

Comment:
The `WordsConfig` class doesn't seem to be very useful. You should be able to deserialize your file directly to a `List<string>`.

Reply:
Also make that class explicitly public or internal. (I know I forgot that on the `AcronymLinker` class, which I'll fix now).